docs/Installation.md

```diff
@@ -27,31 +28,51 @@ To run it as a standalone process you only need to run the binary file downloade
 
 ### Installation using script
 
+1. Get your [installation token][sumologic_docs_install_token] if you don't have it already and assign it to environment variable:
+
+   ```bash
+   export SUMOLOGIC_INSTALL_TOKEN=<TOKEN>
+   ```
+
 1. Run installation script:
 
     ```bash
-    bash <(curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh)
+    bash <(curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh) --installation-token "${SUMOLOGIC_INSTALL_TOKEN}"
     ```
 
-    It is going to perform install or upgrade operation by placing the latest version in `/usr/local/bin`,
+    It is going to perform the following operations:
 
-1. [Verify the installation](#verify-the-installation)
+      - install or upgrade operation by placing the latest version as `/usr/local/bin/otelcol-sumo`,
+      - get [static configuration](../examples/sumologic.yaml) and place it as `/etc/otelcol-sumo/sumologic.yaml`
+      - create user configuration directory (`/etc/otelcol-sumo/conf.d`) with `common.yaml` file which will contain installation token
+      - for Systemd:
 
-1. Prepare the configuration according to [this](Configuration.md) document and save it in `config.yaml`.
+        - the script is going to get [Systemd service configuration](../examples/systemd/otelcol-sumo.service) and place it as `/etc/systemd/system/otelcol-sumo.service`
+        - create a `opentelemetry` user and group which will be used to run the service
```

Comment:
I wonder why this user is called `opentelemetry` and not `otelcol-sumo`? 🤔

Reply:
This is how we did for systemd before installation script

Reply:
Upstream calls this user `otel`: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/buildscripts/packaging/fpm/otel-contrib-collector.service#L11-L12

I'd call it `otelcol-sumo` for consistency.

Reply:
Ok, will change it in another PR as it affects more than install script

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/pull/765